aztfq/aztfq_test.go

```diff
@@ -170,6 +183,146 @@ func TestBuildLookupTable(t *testing.T) {
 						PropertyAddr: "/p2",
 					},
 				},
+				"properties.array.*.test": {
+					{
+						ResourceType: "azurerm_bar",
+						PropertyAddr: "/array",
+					},
+				},
+			},
+		},
+	}, table)
+}
+
+func TestBuildLookupTable_removeArraySymbol(t *testing.T) {
```

Comment:
Instead of replicating the above test case, can we focus on the explict/implicit array index?